src/optimagic/deprecations.py

```diff
@@ -95,3 +96,33 @@ def replace_and_warn_about_deprecated_algo_options(algo_options):
         out[replacements[k]] = algo_options[k]
 
     return out
+
+
+def replace_and_warn_about_deprecated_bounds(
+    lower_bounds,
+    upper_bounds,
+    bounds,
+    soft_lower_bounds=None,
+    soft_upper_bounds=None,
+):
+    old_bounds = {
+        "lower": lower_bounds,
+        "upper": upper_bounds,
+        "soft_lower": soft_lower_bounds,
+        "soft_upper": soft_upper_bounds,
```

Comment:
Should be `"lower_bounds"`, `"upper_bounds"`, `"soft_lower_bounds"`, ..., right?

Reply:
No, but I need to adjust for it in the error message.